src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -1,58 +1,109 @@
 import SectionLabel from '@bbc/psammead-section-label';
+import SkipLinkWrapper from '#components/SkipLinkWrapper';
 
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import useToggle from '#hooks/useToggle';
-import CpsOnwardJourney from '../CpsOnwardJourney';
 import { GridItemMediumNoMargin } from '#app/components/Grid';
-import RecommendationsPromo from './RecommendationsPromo';
 import RecommendationsPromoList from './RecommendationsPromoList';
+import RecommendationsPromo from './RecommendationsPromo';
+
+const Wrapper = styled.div`
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING};
+  }
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+const SkipWrapper = ({ skipLink, service, children }) => {
+  return (
+    <SkipLinkWrapper service={service} {...skipLink}>
+      {children}
+    </SkipLinkWrapper>
+  );
+};
+
+const skipLinkPropShape = {
+  terms: shape({
+    '%title%': string,
+  }),
+  endTextVisuallyHidden: string,
+  endTextId: string,
+  text: string,
+};
+
+SkipWrapper.propTypes = {
+  service: string.isRequired,
+  children: node.isRequired,
+  skipLink: shape(skipLinkPropShape),
+};
+
+SkipWrapper.defaultProps = {
+  skipLink: null,
+};
 
 const RecommendationsWrapper = styled.div`
-  background-color: ${C_LUNAR};
+  background-color: ${C_GHOST};
   margin: ${GEL_SPACING_TRPL} 0;
-  padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
+  padding: ${GEL_SPACING_DBL} 0;
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
     margin: 0 0 ${GEL_SPACING_TRPL};
     padding: 0 0 ${GEL_SPACING_DBL};
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     margin: 0 0 ${GEL_SPACING_TRPL};
-    padding: ${GEL_SPACING_DBL} 0;
-  }
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    padding: ${GEL_SPACING_DBL};
   }
 `;
 
 const LabelComponent = styled(SectionLabel)`
   margin: 0;
   padding: 0;
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    margin: 0 0 ${GEL_SPACING_DBL};
+    margin: 0 0 ${GEL_SPACING};
   }
 `;
 
-const CpsRecommendations = ({ items, parentColumns }) => {
-  const { recommendations, translations } = useContext(ServiceContext);
+const CpsRecommendations = ({ items }) => {
+  const { recommendations, translations, script, service, dir } = useContext(
+    ServiceContext,
+  );
   const { enabled } = useToggle('cpsRecommendations');
+  const labelId = 'recommendations-heading';
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': labelId,
+  };
+
+  const CpsOnwardJourneyWrapper = ({ children }) => (
+    <Wrapper data-e2e={labelId} {...a11yAttributes}>
+      {children}
+    </Wrapper>
+  );
+
+  CpsOnwardJourneyWrapper.propTypes = {
+    children: node.isRequired,
+  };
```

Comment:
Is this wrapper necessary now? Could we apply the attributes to `RecommendationsWrapper`?  I think it was used to fix a weird bug when we re-used story promo but it might not be present now. Also the naming makes it sound like all onward journey's should be using this wrapper.